### DISCUSSION TITLE
Add support for spacy numerizer

### DISCUSSION THREAD

**Discussion by DraconInteractive** [d99kris/spacy-cpp#22] [Ideas]
Hi there :) The context of this question is that I am using spacy to parse the output of a C++ speech recognition tool I have. Now, its somewhat unpredictable with numbers. If I ask it "set a time for 10 minutes" it could return either numeral or word. 
- "set a timer for 10 minutes"
- "set a timer for ten minutes"

Spacy has an extension called Numerizer (https://spacy.io/universe/project/numerizer) that is essentially a big regex parser for the spacy output to retrieve everything in numerical form. 

Is it possible to integrate the extension with spacy cpp, and/or is there another more common way to achieve this? 
I am not generally a c++ programmer, I mainly use .Net/C# so I'm unfamiliar with many of the popular libraries etc. 

My thanks for your time!

**Comment by d99kris** [comment]
Hi @DraconInteractive - I'm not very well-versed in NLP so I'm not sure what the best practise is. About the question to enable `numerizer` access through `spacy-cpp`, intuitively I think it should be feasible, although I haven't looked into it properly yet. Will see if I can find some time to check and add it, will update here again if/when I've tried it, but it might take a while.

**Reply by DraconInteractive** [reply]
Thanks! I have created a temporary solution that I 'think' works, just by using the various combinations of digits, 'tens' (ten, twenty, etc) and hundreds/thousands but I am definitely interested to see if a more integral solution can be found

**Comment by d99kris** [comment]
I don't anticipate getting a chance to look into this anytime soon. I will close this request for now, but if anyone like to implement support, a PR will be considered.
